app/src/main/res/values/strings.xml

```diff
@@ -20,17 +20,17 @@
     <!-- Long click options -->
     <string name="long_click_title">What do you want to do?</string>
     <string name="long_click_open">Open \"%1s\"</string>
-    <string name="long_click_store">See its store page</string>
+    <string name="long_click_store">View store page</string>
     <string name="long_click_settings">Access its settings</string>
     <string name="long_click_remove_shortcut">Remove this shortcut</string>
 
     <!-- About -->
     <string name="app_name">Discreet Launcher</string>
     <string name="author" translatable="false">Vincent Falzon</string>
-    <string name="licence_name">GNU General Public License v3.0</string>
+    <string name="licence_name">GPLv3+</string>
     <string name="license_link" translatable="false">https://gnu.org/licenses/gpl-3.0.txt</string>
-    <string name="website_description">To access the source code, report an issue or see my other projects, visit the website</string>
+    <string name="website_description">Improve the source code, other projects, and report issues on</string>
     <string name="website_link" translatable="false">https://vincent-falzon.com</string>
-    <string name="translation_credit"/>

```

Comment:
Not sure if this should be a string or not. Optional if translators want to mention themselves? is <e-mail> supported btw?

Reply:
*(As discussed in PR #77, maybe I will add a comment template here to make it clearer. For now, I simply added an explanation about it in the Contributing file)*

Both `<string name="translation_credit"/>` and `<string name="translation_credit"></string>` are valid in XML but the first one is usually preferred when the body is empty.
The second one even generates a warning in Android Studio, which is why I use the first.

Reply:
Interesting. Reverting then.